CampaignConfig/mdc2020_beam.cfg

```diff
@@ -1,5 +1,5 @@
 [global]
-includes = /mu2e/app/users/srsoleti/workDir/Production/CampaignConfig/mdc2020_prolog.cfg
+includes = /cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020u/Production/CampaignConfig/mdc2020_prolog.cfg
```

Comment:
Is there a way to avoid hardcoding MDC2020u and making it a parameter.  For example does POMS support the concept of an include path?

Reply:
Yes, there is. Let me try to fix this in my next round, I noticed a few other things